homeassistant/components/vallox/__init__.py

```diff
@@ -114,16 +121,34 @@ def get_metric(self, metric_key: str) -> StateType:
 
         return value
 
+    def get_model(self) -> str | None:
+        """Return the model, if any."""
+        model = _api_get_model(self.metric_cache)
+
+        if not isinstance(model, str) or model == "Unknown":
+            return None
+
+        return model
+
+    def get_sw_version(self) -> str | None:
+        """Return the SW version."""
+        sw_version = _api_get_sw_version(self.metric_cache)
+
+        if not isinstance(sw_version, str):
```

Comment:
Please use cast() or adhere to https://www.python.org/dev/peps/pep-0484/, if you only use `str()` for typing purposes.

Reply:
What would it be other than a string?  If its `None`, its much faster to check if its `is None` than an isinstance.

Reply:
I checked the 3rd party library again, it can only be `str`, so I replaced this with a cast.